core/saltfilter/bloomring.go

```diff
@@ -0,0 +1,66 @@
+package saltfilter
```

Comment:
This package should probably be an internal one, as it's implementation details that should not be exported.

Reply:
And BloomRing keep as a public struct to provide for whole repo use